src/utils/updateNickname.ts

```diff
@@ -0,0 +1,28 @@
+import { INTERNAL_SERVER_ERROR } from "../constants/responses";
+import { DISCORD_BASE_URL } from "../constants/urls";
+import { env } from "../typeDefinitions/default.types";
+
+export async function updateNickName(
+  discordId: string,
+  nickname: string,
+  env: env
+) {
+  const changeNickNameURL = `${DISCORD_BASE_URL}/guilds/${env.DISCORD_GUILD_ID}/members/${discordId}`;
```

Comment:
can this URL be moved to a constants file and its casing be changed to uppercase snake cased letters?

Reply:
This URL is formed from user input how can it be a constant? it's different for every user. As of what I know Constants are not re-assignable after initialization. Could you point me somewhere where it says something different?
I have used a part of the URL from constants i.e `DISCORD_BASE_URL`
If it is not in the constants file do we need the uppercase snake case?